src/Stratis.Bitcoin.Features.SignalR/Broadcasters/StakingBroadcaster.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Connection;
+using Stratis.Bitcoin.Features.Miner.Interfaces;
+using Stratis.Bitcoin.Features.SignalR.Events;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+using Stratis.Bitcoin.Signals;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.SignalR.Broadcasters
+{
+    public class StakingBroadcaster : ClientBroadcasterBase
```

Comment:
We comment all classes with a /// summary. So just add a short comment on what this broadcaster is used for (even if its obvious)